app/views/foods/_form.html.erb

```diff
@@ -1,18 +1,20 @@
-<h2>Add new food:</h2>
-<%= form_with model: @food do |form| %>
-  <p>
-    <%= form.label :name %> <br>
-    <%= form.text_field :name %>
-  </p>
+<div class="container d-flex flex-column align-items-center">
+  <h2 class="align-items-stretch">Add new food</h2>
```

Comment:
[OPTIONAL] Is the class `align-items-stretch` working here? This property is only for flex containers and I don't think h2 is one, correct me if I'm wrong but if that's not the case, kindly remove it

Reply:
You are right. It is a left over from earlier experimenting. :+1: